ui/buildServer.sh

```diff
@@ -8,8 +8,6 @@ fi
 cd ..
```

Comment:
maybe we want to get rid of this if altogether?

Reply:
In future we can use it ;) I will leave it.